shell/browser/electron_permission_manager.cc

```diff
@@ -72,6 +75,9 @@ class ElectronPermissionManager::PendingRequest {
         content::ChildProcessSecurityPolicy::GetInstance()
             ->GrantSendMidiSysExMessage(render_frame_host_id_.child_id);
       } else if (permission == blink::PermissionType::GEOLOCATION) {
+#if BUILDFLAG(IS_MAC)
+        InitializeGeolocation();
```

Comment:
This is only called from a single file - we don't need to create a helper file. We can just move the relevant code into the anonymous namespace and call that.